PYME/LMVis/Extras/pointwiseColoc.py

```diff
@@ -21,8 +21,18 @@ def OnPointwiseColoc(self, event):
         #from PYME import mProfile
         #mProfile.profileOn(['distColoc.py'])
         from PYME.Analysis.Colocalisation import distColoc
+        from PYME.DSView.modules.coloc import ColocSettingsDialog
+
+        chans = self.visFr.pipeline.colourFilter.getColourChans()[::1]
+
+        dlg = ColocSettingsDialog(self.visFr, names=chans, show_bins=False)
+        dlg.ShowModal()
```

Comment:
Check return and abort if user cancelled

Reply:
Weirdly returns `ID_OK` if you click `OK` or close the dialog...